lte/gateway/python/integ_tests/s1aptests/test_attach_mobile_reachability_timer_expiry.py

```diff
@@ -46,43 +60,58 @@ def test_mobile_reachability_timer_expiry_handler(self):
         # context release
         time.sleep(0.5)
 
-        print("************************* Sending UE context release request ",
-              "for UE id ", ue_id)
+        print(
+            "************************* Sending UE context release request ",
+            "for UE id ",
+            ue_id,
+        )
         # Send UE context release request to move UE to idle mode
         req = s1ap_types.ueCntxtRelReq_t()
         req.ue_Id = ue_id
         req.cause.causeVal = gpp_types.CauseRadioNetwork.USER_INACTIVITY.value
         self._s1ap_wrapper.s1_util.issue_cmd(
-            s1ap_types.tfwCmd.UE_CNTXT_REL_REQUEST, req)
+            s1ap_types.tfwCmd.UE_CNTXT_REL_REQUEST, req
+        )
         response = self._s1ap_wrapper.s1_util.get_response()
         self.assertEqual(
-            response.msg_type, s1ap_types.tfwCmd.UE_CTX_REL_IND.value)
-
-        print("************************* Sending Service request for UE id ",
-              ue_id)
-        # Delay by 6minutes to ensure Mobile reachability timer expires.
-        # By default, mobile reachability timer value configured to 54 mins in
-        # mme.conf.template file.For testing purpose manually reduce the mobile
-        # reachability time to 1minute in mme.conf.template file
-        # MOBILE REACHABILITY TIMER VALUE = 1minutes(conf file) + delta value
-        time.sleep(360)
+            response.msg_type, s1ap_types.tfwCmd.UE_CTX_REL_IND.value
+        )
+
+        # Delay by 6 minutes to ensure Mobile reachability timer expires.
+        # MOBILE REACHABILITY TIMER VALUE = 1 minutes(conf file) + delta value
```

Comment:
1 minutes(conf file) -> 1 minute (conf file)

Reply:
Corrected the comment and updated the PR